src/styles/utils/_variables.scss

```
@@ -0,0 +1 @@
+$duration-for-transition: 300ms;
```

I guess this way it is good enough
```suggestion
$transition-duration: 300ms;
```